### PR TITLE
Auth rating

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
# Login Page

[CSB Link](https://enzwun.csb.app/login)
Test Credentials are 
### ```password : adarshBalika123```

# Sign Up Page

[CSB Link](https://enzwun.csb.app/signup)

# Rating Feature 
[CSB Link](https://enzwun.csb.app/product)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -20,6 +21,7 @@ function App() {
           <Route path="/product/:productId" element={<SingleProductPage />} />
           <Route path="/product" element={<Product />} />
           <Route path="/cart" element={<Cart />} />
+          <Route path="/mockman" element={<Mockman />} />
         </Routes>
       </BrowserRouter>
```
Comment (AtulPant2704) [inline src/App.js]: You should have your `BrowserRouter` in your index.js, here you should just have Routes.

File: src/index.js [hunk 2]
```diff
@@ -4,14 +4,20 @@ import './index.css';
...
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
     <ProductProvider>
-      <App />
+      <FilterProvider>
+        <AuthProvider>
+          <App />
+        </AuthProvider>
```
Comment (AtulPant2704) [inline src/index.js]: Your `AuthProvider` should be at the top above `FilterProvider` and `ProductProvider.`

File: src/pages/Login/Login.jsx [hunk 3]
```diff
@@ -2,20 +2,43 @@ import React from 'react';
 import { useState } from 'react';
 import Navbar from '../../components/Navbar/Navbar';
 import './Login.css';
+import { useAuth } from '../../context/auth/authContext';
+import axios from 'axios';
+// import Toast from '../../components/Toast/Toast';
```
Comment (AtulPant2704) [inline src/pages/Login/Login.jsx]: You can remove this comment.

File: src/util/getRating.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+export function getRatingList({ rating }, list) {
```
Comment (dipteekhd) [inline src/util/getRating.js]: can use arrow function here!

File: src/pages/Signup/Singup.jsx [hunk 5]
```diff
@@ -3,13 +3,18 @@ import Navbar from '../../components/Navbar/Navbar';
 import { useState } from 'react';
 import '../../pages/Login/Login.css';
 import './Signup.css';
+import axios from 'axios';
+import { useAuth } from './../../context/auth/authContext';
+import { useNavigate, Link } from 'react-router-dom';
 function Singup() {
```
Comment (dipteekhd) [inline src/pages/Signup/Singup.jsx:9]: can use arrow function here!

**Review by dipteekhd — APPROVED** [review]
use arrow function in project

**Review by AtulPant2704 — APPROVED** [review]

**Pull request merged**